docs/mapping_file_based_mapping.md

```diff
@@ -52,6 +52,18 @@ A typical entry looks like this:
 }
 ```
 folio_field and legacy_field are mandatory. All other fields are optional. 
+### The pirority of the mappings in the mapping entry:
+There might be exceptions to this in some areas, but generally, this is the how the mapping works:
+1. If there are reference data mappings or special cases for particular fields, then this has precedence
+2. Values added to the *value* field are returned immediately without any further manipulation
```

Comment:
Does `value` override the reference data mapping? I don't think I've ever tried it…

Part of me feels like there should be some nesting with "or"s sprinkled throughout here…